docs/static/ls-ls-http.asciidoc

```diff
@@ -8,3 +8,126 @@ NOTE: Check out these <<http-considerations,considerations>> before you implemen
 
 For now, <<plugins-outputs-http,http output>> to <<plugins-inputs-http,http input>> with manual configuration may be the best path forward if these limitations don't apply to your use case.
 
+==== Configuration overview
+
+To use the HTTP protocol to connect two Logstash instance you will:
+
+. Configure the downstream (server) Logstash to use HTTP input
+. Configure the upstream (client) Logstash to use HTTP ouput
+. Securing the communication between HTTP input and HTTP output
+
+[[configure-downstream-logstash-http-input]]
+===== Configure the downstream Logstash to use HTTP input
+
+To configure the downstream Logstash (the receiving Logstash) you need to configure the HTTP input to receive connections. The minimum recommended configuration requires you to set the following options:
+
+* `port` - to set a custom port.
+* `additional_codecs` - To set `application/json` to be `json_lines`.
+
+[source,json]
+----
+input {
+    http {
+        port => 8080
+        additional_codecs => { "application/json" => "json_lines" }
+    }
+}
+----
+
+[[configure-upstream-logstash-http-output]]
+===== Configure the upstream Logstash to use HTTP output
+
+In order to optain the best performance when sending data from one Logstash to another, it needs to be batched and compressed. As such, the upstream Logstash (the sending Logstash) needs to be configured with the following options:
+
+* `url` - The receiving Logstash.
+* `http_method` - Set to `post`.
+* `retry_non_idempotent` - Set to `true`, in order to retry failed events.
+* `format` - Set to `json_batch` to batch the data.
+* `http_compression` - Set to `true` ensures the data is compressed.
+
+[source,json]
+----
+output {
+    http {
+        url => '<protocol>://<downstream-logstash>:<port>'
+        http_method => post
+        retry_non_idempotent => true
+        format => json_batch
+        http_compression => true
+    }
+}
+----
+
+[[securing-logstash-to-logstash]]
+===== Securing Logstash to Logstash
```

Comment:
```suggestion
===== Secure Logstash to Logstash
```

Reply:
Should match the tense of previous sections.  If you are concerned that "secure" might be initially read as an adjective, you could try "Secure your LS to LS connection" or just "Secure your connection."